resources/views/discord/speedtest-completed.blade.php

```diff
@@ -7,3 +7,5 @@
 - **Ping:** {{ $ping }}
 - **Download:** {{ $download }}
 - **Upload:** {{ $upload }}
+- **Packet Loss:** {{ $packetLoss }} **%**
+- **URL:** {{ $url }}
```

Comment:
Can you add an empty line at the end of the file here?

Reply:
Done! Just for my understanding why is the line needed?